cluster/operations/tls-vars.yml

```diff
@@ -14,5 +14,5 @@
     type: certificate
     options:
       ca: atc_ca
-      common_name: ((external_host))
       alternative_names: [((external_host))]
+      organization: atcOrg
```

Comment:
Is this related/necessary?

Reply:
You need an organization or common name for credhub to generate the variable I think.